README.md

```diff
@@ -141,34 +141,44 @@ List<Project> allProjects = projectPager.all();
 
 ---
 ## Java 8 Stream Support
-As of GitLab4J-API 4.9.2, you can also stream list based items in a Java 8 Stream using a Pager instance as follows:
+As of GitLab4J-API 4.9.2, all GitLabJ-API methods that return a List result also similarlly named method returns a Java 8 Stream.  The Stream returning methods use the following naming convention:  ```getXxxxxStream()```.  For example:
+
 ```java
-// Get a Pager instance to get a Stream<Project> instance.
-Pager<Project> projectPager = gitlabApi.getProjectsApi().getProjects(10);
+// List method
+List<Project> projects = gitlabApi.getProjectApi().getProjects();
 
-// Stream the Projects printing out the project name.
-projectPager.stream().map(Project::getName).forEach(name -> System.out.println(name));
-```
-The following API classes also include ```getXxxxxStream()``` methods which return a Java 8 Stream:
-```
-GroupApi
-IssuesApi
-NotesApi
-ProjectApi
-RepositoryApi
-TagsApi
-UserApi
+// Stream method
+Stream projectsStream<Project> = gitlabApi.getProjectApi().getProjectsStream();
 ```
+
 Example usage:
+
 ```java
 // Stream the visible Projects printing out the project name.
-gitlabApi.getProjectsApi().getProjectsStream().map(Project::getName).forEach(name -> System.out.println(name));
+Stream projectsStream<Project> = gitlabApi.getProjectApi().getProjectsStream();
+projectsStream.map(Project::getName).forEach(name -> System.out.println(name));
 
 // Operate on the stream in parallel, this example sorts User instances by username
```

Comment:
I'm not sure if this is a good idea to give such an example here. Parallel operations are generally used for performance optimization, but here project gathering is so slow, that this operation doesn't add any improvement. As a result, the user can be confused that this operation can be super fast and this library can fetch data in multiple channels.

Reply:
Parallel operations are often performed after fetching the data, paralell operations are not only used to speed up fetching, I have added more info to make the fact that reading of the data for the Stream cannot done in paralell.